src/node/test/encryptor.cpp

```diff
@@ -67,6 +69,7 @@ TEST_CASE("Additional data")
   // Setting 1 ledger secret, valid for version 1+
   uint64_t node_id = 0;
   auto secrets = std::make_shared<ccf::LedgerSecrets>();
+  secrets->set_secret(1, std::vector<uint8_t>(16, 0x42));
```

Comment:
If these secrets are unused in these unit tests, can we pass an empty vector or at least a named value (`dummy_secrets` or similar)?

Reply:
This is actually used as `secrets` is passed to the `encryptor` which uses these secrets for encryption/decryption.